includes/class-newspack.php

```diff
@@ -335,6 +335,9 @@ public function admin_redirects() {
 			return;
 		}
 		delete_transient( NEWSPACK_ACTIVATION_TRANSIENT );
+		if ( \get_option( NEWSPACK_SETUP_COMPLETE, false ) ) {
+			return false;
```

Comment:
```suggestion
			return;
```

NIT, no need to return a value on this action hook.